pkg/synopsysctl/cmd_create.go

```diff
@@ -623,6 +623,15 @@ var createPolarisCmd = &cobra.Command{
 			}
 		}
 
+		versions, err := polaris.GetVersions(baseURL)
```

Comment:
Can we have the code to call getversion and IsInStringSlice as a method somewhere in synopsysctl and call the method here?

Reply:
Done